lib/minitest/reporters/base_reporter.rb

```diff
@@ -89,8 +89,12 @@ def print_info(e, name=true)
         print "#{e.exception.class.to_s}: " if name
```

Comment:
all types of output from test results is padded.. might wanna use `print_with_info_padding` here too